README.md

```diff
@@ -112,6 +112,19 @@ It starts etcd and simulator-server locally.
 
 To run the frontend, please see [README.md](web/README.md) on ./web dir.
 
+## [Beta] Existing cluster importing
+
+**PLEASE AVOID** using it in clusters with large amounts of resources.
+The simulator will try to import ALL resources related to scheduling from your cluster and it may badly affect your cluster's api-server.
```

Comment:
Sorry, I know it's suggested by the past me, but now I know it isn't needed to be mentioned.  

```suggestion
```